src/app/features/edit-task/edit-task.component.html

```diff
@@ -0,0 +1,34 @@
+<form [formGroup]="currentTask" (ngSubmit)="onUpdate()">
+
+  <div class="form-layout">
+    <div>
+      <label for="title">Title:</label>
+      <input id="title" type="text" formControlName="title" required>
+      <app-input-validator [property]="title" [minLength]="5"></app-input-validator>
```

Comment:
take into consideration that task-details was a future task, but it was done now, as a result of that you've created a separate page for editing a task, when the requirements of that ticket were to allow the task to be editable in the task details page.